Change char id to enum bittorrent_message_id id to prevent FTBFS on
powerpc and s390

diff --git a/src/protocol/bittorrent/common.h b/src/protocol/bittorrent/common.h
index 9d44b76..a656cfd 100644
--- a/src/protocol/bittorrent/common.h
+++ b/src/protocol/bittorrent/common.h
@@ -110,7 +110,7 @@ struct bittorrent_peer_request {
 
 	/* This holds the message id when the request struct is used for queuing
 	 * pending messages. */
-	char id;			/**< @-> enum bittorrent_message_id */
+	enum bittorrent_message_id id;	/**< @-> enum bittorrent_message_id */
 
 	unsigned int cloned:1;		/**< The request was cloned. */
 	unsigned int requested:1;	/**< Whether it has been requested. */
